satellite/notification/endpoint.go

```diff
@@ -0,0 +1,39 @@
+package notification
+
+import (
+	"context"
+
+	"storj.io/storj/pkg/pb"
+)
+
+type Endpoint struct {
+	service *Service
+}
+
+func NewEndpoint(service *Service) *Endpoint {
+	return &Endpoint{
+		service: service,
+	}
+}
+
+// ProcessNotification process notification by rpc.
+func (endpoint *Endpoint) ProcessNotification(ctx context.Context, message *pb.NotificationMessage) (*pb.NotificationResponse, error) {
```

Comment:
Neat idea with the private GRPC endpoint being used for this but maybe we can rename the function to avoid confusion with the other methods in the service?